parrot.rs

```diff
@@ -0,0 +1,73 @@
+use core::sync::atomic::{AtomicUsize, Ordering};
+use kernel::{
+    bindings::msleep,
```

Comment:
If you have the time, please submit a PR to add `msleep` to the `kernel` crate -- the idea is that, in the future, modules do not use `bindings` directly.